### ISSUE TITLE
Enable automatic queue start

### ISSUE THREAD

**Issue by eltiel** [TheNickOfTime/handbrake-web#247]
### Describe your feature request
To allow queues to start automatically

### Provide additional context
Handbrake Web would be great if i could treat it as a "drop and forget" solution where I could just drop a file in a watched folder and let it do its job. Currently, everything works great down to the automatic job creation but I still need to manually start the queue.

**Comment by TheNickOfTime** [comment]
Yeah perhaps I should change the default behavior to the queue being active rather than stopped. Or possibly add a configuration option for watchers to automatically start the queue. I'll consider this for the v0.8 release cycle!
